libr/include/r_bin_dwarf.h

```diff
@@ -28,8 +28,8 @@ extern "C" {
 #define DW_LNE_set_address              0x02
 #define DW_LNE_define_file              0x03
 #define DW_LNE_set_discriminator        0x04  /* DWARF4 */
-#define DW_LNE_lo_user			0x80
-#define DW_LNE_hi_user			0xff
+#define DW_LNE_lo_user					0x80
```

Comment:
why? I think you should remove all the tabs and just use spaces as all the other #defines above.